docs/_source/tutorials/notebooks/labelling-textclassification-sentencetransformers-weaksupervision.ipynb

```diff
@@ -11,14 +12,15 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "00335193",
    "metadata": {
     "id": "00335193"
    },
    "source": [
     "In this tutorial, we show how weak supervision workflows in Argilla can be extended with sentence embeddings.\n",
-    "We start from the weak supervision workflow presented in our [Weak supervision with Argilla tutorial](./labelling-textclassification-sentencetransformers-weaksupervision.ipynb) and improve on its results by extending the coverage of its rules.\n",
+    "We start from the weak supervision workflow presented in our [Weak supervision with Argilla tutorial](https://docs.argilla.io/en/latest/guides/programmatic_labeling_with_rules.html) and improve on its results by extending the coverage of its rules.\n",
```

Comment:
local references to files, see comment below

Reply:
they should be fixed now!